charts/gateway/tests/configmap_test.yaml

```diff
@@ -0,0 +1,58 @@
+suite: test confimap
+templates:
+  - configmap.yaml
+tests:
+  - it: should have valid kind type and valid LICENSE, HEAP settings
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.ACCEPT_LICENSE
+          value: "false"
+      - equal:
+          path: data.SSG_CLUSTER_HOST
+          value: "my.localdomain"
+      - equal:
+          path: data.SSG_JVM_HEAP
+          value: "2g"
+
+  - it: should have jdbc connection with the default url
+    set:
+      database.create: true
+    asserts:
+      - equal:
+          path: data.SSG_DATABASE_JDBC_URL
+          value: jdbc:mysql://RELEASE-NAME-mysql:3306/ssg
+
+  - it: should not have jdbc connection url
+    set:
+      database.create: false
```

Comment:
For External Database , database.create will be false but jdbcUrl still will be needed